### ISSUE TITLE
[Accessibility]A11y_VSCode_FileIconTheme_Screenreader: Voiceover does not read the menu list items present in the tree view.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52554]
Environment Details:
VSCode Version : 1.24.1
OS Version : MAC OS HighSierra10.13.5

Additional Details:
Voiceover,MAS1.3.1

Repro Steps:
1)Launch VSCode.
2)Navigate to Settings and select "Select File Icon Theme" button.
3)Check with Voiceover is reading.

Actual:
Voiceover does not read the menu list present on the screen.

Expected:
Voiceover should read the menu list which is present on the screen.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)

User Impact:
User will not able to know that there are menu list items present on the screen.

MAS Reference:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
](url)

Attachment for Reference:
[[Accessibility]A11y_VSCode_FileIconTheme_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2122477/Accessibility.A11y_VSCode_FileIconTheme_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52383
